packaging/windows/WiXTools/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilationOptions": {
+      "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "WiX": "3.10.0.2103-pre1"
```

Comment:
Does it work if the %path% is also modified after you install to have other things after it on the path?